src/renderer/status-pill/pill.css

```
@@ -0,0 +1,647 @@
+
+.perm-summary {
+  margin: 0 0 10px 0;
+  padding: 8px 10px;
+  background: var(--diff-bg);
+  border: 1px solid var(--border);
+  border-radius: 6px;
+  font-family: 'Geist', ui-monospace, 'SF Mono', Menlo, monospace;
+  font-size: 11px;
+  color: var(--foreground);
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)